tests/foreman/api/test_hostgroup.py

```diff
@@ -2,6 +2,7 @@
 """Tests for the ``hostgroups`` paths."""
 from fauxfactory import gen_string
 from nailgun import client, entities, entity_fields
+from nailgun.entity_mixins import _get_entity_id
```

Comment:
I'd like to go ahead and do this, then refactor the code in the future. It's true that we're taking on technical debt with this approach, but I think the technical debt is appropriate here. Between yesterday and today, I'm submitting three different sets of API tests that all have a nearly identical form. Adding these tests now and refactoring later ensures that the refactor is general enough to work with all three tests.

My idea for refactoring this is to add a simple function in to Robottelo that, given a name, generates all the permutations Satellite might return. I can then refactor the tests to have assertions roughly like this:

```python
        self.assertTrue(
            any([attr in self.hg_attrs.keys() for attr in one_to_one_names('content_source')]),
            'None of {0} are in {1}'.format(attrs, self.hg_attrs.keys())
        )
```